foundry.toml

```diff
@@ -5,6 +5,7 @@ libs = ["node_modules", "lib"]
 gas_limit = "18446744073709551615"
 memory_limit = 2147483648
 verbosity = 3
+evm_version = "cancun"
```

Comment:
would this be an issue for people using prev evm versions and importing this into their repo?

Reply:
It's just to support tstore checks, if they use < versions it should still be fine (only the test actually use tstore verbatim)